src/textures/Texture.js

```diff
@@ -170,6 +170,29 @@ Texture.prototype = Object.assign( Object.create( EventDispatcher.prototype ), {
 
 		}
 
+		function addImage( image ) {
+
+			// TODO: Move to THREE.Image
+
+			if ( image.uuid === undefined ) {
+
+				image.uuid = _Math.generateUUID(); // UGH
```

Comment:
Off topic: I wanna remove this pollution.

Reply:
This will be solved via #17949.